src/nsrt_learning/strips_learning/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""This directory contains algorithms for STRIPS operator learning."""
+
+from typing import List, Set, Type
+
+from predicators.src.nsrt_learning.strips_learning.base_strips_learner import \
+    BaseSTRIPSLearner
+from predicators.src.nsrt_learning.strips_learning.clustering_learner import \
+    ClusterAndIntersectSidelineHarmlessnessSTRIPSLearner, \
+    ClusterAndIntersectSidelinePredictionErrorSTRIPSLearner, \
+    ClusterAndIntersectSTRIPSLearner, ClusterAndSearchSTRIPSLearner
+from predicators.src.nsrt_learning.strips_learning.gen_to_spec_learner import \
+    BackchainingSTRIPSLearner
+from predicators.src.nsrt_learning.strips_learning.oracle_learner import \
+    OracleSTRIPSLearner
+from predicators.src.settings import CFG
+from predicators.src.structs import LowLevelTrajectory, \
+    PartialNSRTAndDatastore, Predicate, Segment, Task
+
+
+def learn_strips_operators(
```

Comment:
can we do the same pattern as envs and approaches here? i like it... also, maybe there's a chance it would fix the circular import issue